.github-labelsrc.json

```diff
@@ -172,5 +172,11 @@
     "name": "🧟 Type: Technical Debt",
     "color": "d876e3",
     "description": ""
+  },
+  {
+    "id": 1469909491,
```

Comment:
Got this after creating the label from the `data-label-id` in the DOM of GitHub.

Reply:
You should have used the CLI, but anyway...

Reply:
What CLI? Where can I read up on that?

Reply:
https://github.com/commercetools/github-labels/